helm-charts/basehub/values.yaml

```diff
@@ -45,6 +56,12 @@ binderhub-service:
       node_selector:
         # Schedule builder pods to run on user nodes only
         hub.jupyter.org/node-purpose: user
+      # memory_request are currently zeroed based on discussion in:
+      # https://github.com/2i2c-org/infrastructure/issues/4371#issuecomment-2223168854
+      memory_request: 0 # 1G in mybinder.org-deploy
```

Comment:
The build pods aren't daemonset pods, so they won't cause as much trouble and could possibly have a request. They need to be scheduled on a node with a daemonset pod though, but it felt safer to just go 0 here to avoid complexity.